build.zig.zon

```diff
@@ -1,7 +1,7 @@
 .{
     .name = "lsp-codegen",
     .version = "0.1.0",
-    .minimum_zig_version = "0.12.0",
+    .minimum_zig_version = "0.14.0-dev.1346+31fef6f11",
```

Comment:
Compatibility with the latest tagged release is a nice thing to keep. See the other review comments.